qiita_pet/templates/admin_processing_job.html

```diff
@@ -2,6 +2,116 @@
 
 {% block head %}
 <script type="text/javascript">
+  var jobsTable;
+
+  // This modal view is used to display job details when a job has errored or
+  // succeeded. Needs to be in the main scope because this function is called
+  // from the "onclick" attribute of some elements.
+  var populateModal = function(title, main) {
+    $('#job-output-modal').find('[name="modal-title"]').html(title);
+    $('#job-output-modal').find('[name="modal-main-text"]').html(unescape(decodeURIComponent(main)));
+  }
+
+  // Initialize the previous jobs table once the document is ready
+  $(function() {
+    var renderJobArguments = function(data, type, row, meta) {
+      // select how you want to render the data
+      var lines = [], container;
+
+      for (key in data) {
+        lines.push('<b>' + key + '</b>:');
+        if (typeof data[key] === 'string'){
+          lines.push(data[key]);
+        }
+        else if (typeof data[key] === 'object') {
+          container = data[key];
+
+          // if we have all these attributes then render a named hyperlink to
+          //download the data
+          if (container['body'] !== undefined &&
+              container['filename'] !== undefined &&
+              container['content_type'] !== undefined) {
+              lines.push('<a download="' + container['filename'] +
+                         '" href="data:text/plain;charset=utf-8,' +
+                         encodeURIComponent(container['body']) + '">' +
+                         container['filename'] + '</a>');
+          }
+          else {
+            lines.push('JavaScript Object')
+          }
+        }
+        else {
+          lines.push(data[key]);
+        }
+      }
+
+      return lines.join('<br>');
+    }
+
+    var renderRichDescription = function(data, type, row, meta) {
+      var out = [], status = row[2];
+      var statusToClass = {
+        'error': 'text-danger',
+        'success': 'text-success',
+        'queued': 'text-muted',
+        'running': 'text-info'
+      };
+
+      out.push('<b class="' + statusToClass[status] + '">' + status +
+               '</b>: ');
+
+      if (status === 'running' || status === 'queued') {
+        out.push(row[0] + ' <i>' + row[3] + '</i>')
+      }
+      else {
+        // We write a callback attribute on the link to be able to display a
+        // modal window with the additional success/error details. Note, the
+        // quotation is a bit hard to follow but all in all it's ensuring any
+        // text we get back from the server is properly formatted for
+        // inclusion in an HTML attribute.
+        var callback = 'populateModal("' + row[0] + " completed on " +
+                       row[6] + '",';
+        out.push('<a href="#" data-toggle="modal" data-target="#job-output-modal" onclick=\'' + callback)
+
+        if (status === 'success') {
+          out.push('"Render HTML for Artifact, check with Antonio")\'');
```

Comment:
The info is displayed based on the output artifacts (job.outputs, it's a dict) and using the `display_artifact_info` function in this code.

Reply:
it's working now!